backend/src/main/java/com/google/sps/notifiers/Notifier.java

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main.java.com.google.sps.notifiers;
+
+public class Notifier {
```

Comment:
Do you envision more subclasses of Notifier besides EmailNotifier going forward?

Reply:
I would like to potentially like to capability of creating more types of notifications in the future but as of now I decided to solely restrict it to email. Also, I wanted to use polymorphism within the code in the case it would like to be enhanced

Reply:
Understood. In that case there should be a method on the base class to perform the notification that all the extenders implement.

Reply:
Okay, in that case would it be fine to leave the body of that function blank ?

Reply:
It seems like this class could be abstract and then the method could be abstract as well. Do you see a scenario in which a caller would instantiate Notifier directly instead of one of the subclasses?

Reply:
Made abstract !